mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -35,5 +36,8 @@ def generate_refund_file(request):
     client = api_client.get_connection(request)
     client.bank_admin.transactions.patch(refunded_transactions)
 
+    # create file record
+    post_new_file(request, 'ACCESSPAY', [t['id'] for t in refunded_transactions])
```

Comment:
I kinda worry that we're leaking implementation details between layers of the system here: `ACCESSPAY` and `ADI` are filetypes that users of the bank admin tool user because of the currently-existing supplier contracts for particular pieces of admin infrastructure.

Does the API/DB need to know it's an ADI or AccessPay file, or perhaps there's a more generic term we can use for these groupings of transactions?